internal/xds/rbac/rbac_engine.go

```diff
@@ -29,6 +29,7 @@ import (
 	"net"
 	"strconv"
 
+	v1typepb "github.com/cncf/xds/go/udpa/type/v1"
```

Comment:
#6238 standardized the import names. Please change it accordingly. Same for the test file.

Reply:
Renamed per that PRs standards